admin/src/app/auth.service.ts

```diff
@@ -61,10 +61,21 @@ export class AuthService {
   }
   
   public register(email: string, password: string): Observable<string> {
-    return this.http.post<string>(
-      `${this.apiUrl}api/auth/registration`,
-      { email, password }
-    );
+    const options: Object = {
+      responseType: 'text'
+    };
+    
+    return this.http
+      .post<string>(
+        `${this.apiUrl}api/auth/registration`,
+        { email, password },
+        options
+      )
+      .pipe(
+        tap((token: string) => {
+          localStorage.setItem(ACCESS_TOKEN_KEY, token);
+        })
```

Comment:
I should not change returned from previous method value and I should add token to local storage, it can be called side effect; as I understood, it it the situation "tap" might be used